modules/src/mock_context.rs

```diff
@@ -27,14 +27,23 @@ pub struct MockContext {
     /// A list of `max_history_size` headers, ascending order by their height (latest is last).
     history: Vec<MockHeader>,
 
+    // All the connections in the store.
+    connections: HashMap<ConnectionId, ConnectionEnd>,
+
```

Comment:
can we move these back down again? :)

Reply:
https://github.com/informalsystems/ibc-rs/pull/310/commits/4e552621421472bbafbb10fd93ce9b0b57463b45